setup_linux.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: Flutter Setup for Linux
+permalink: /setup-linux/
+---
+
```

Comment:
Just not at that point yet...good suggestion though!

In these long instructions, the first content a Flutter user interacts adding a TOC makes the page longer! What would help tremendously is some CSS to right-align the TOC and wrap the text around it. Is there an issue for this?